hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/GetCPMembersMessageTask.java

```diff
@@ -0,0 +1,91 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.client.impl.protocol.task.management;
+
+import com.hazelcast.client.impl.protocol.ClientMessage;
+import com.hazelcast.client.impl.protocol.codec.MCGetCPMembersCodec;
+import com.hazelcast.client.impl.protocol.codec.MCGetCPMembersCodec.RequestParameters;
+import com.hazelcast.client.impl.protocol.task.AbstractAsyncMessageTask;
+import com.hazelcast.cp.CPMember;
+import com.hazelcast.cp.CPSubsystemManagementService;
+import com.hazelcast.instance.impl.Node;
+import com.hazelcast.internal.management.ManagementCenterService;
+import com.hazelcast.internal.nio.Connection;
+
+import java.security.Permission;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+public class GetCPMembersMessageTask extends AbstractAsyncMessageTask<RequestParameters, List<SimpleEntry<String, String>>> {
+
+    public GetCPMembersMessageTask(ClientMessage clientMessage, Node node, Connection connection) {
+        super(clientMessage, node, connection);
+    }
+
+    @Override
+    protected CompletableFuture<List<SimpleEntry<String, String>>> processInternal() {
+        CPSubsystemManagementService cpService =
+                nodeEngine.getHazelcastInstance().getCPSubsystem().getCPSubsystemManagementService();
+        return cpService.getCPMembers().toCompletableFuture()
+                .thenApply(cpMembers -> {
+                    List<SimpleEntry<String, String>> result = new ArrayList<>();
+                    for (CPMember cpMember : cpMembers) {
+                        String address = "[" + cpMember.getAddress().getHost() + "]:" + cpMember.getAddress().getPort();
+                        result.add(new SimpleEntry<>(cpMember.getUuid().toString(), address));
+                    }
+                    return result;
+                });
+    }
+
+    @Override
+    protected RequestParameters decodeClientMessage(ClientMessage clientMessage) {
+        return MCGetCPMembersCodec.decodeRequest(clientMessage);
+    }
+
+    @Override
+    protected ClientMessage encodeResponse(Object response) {
+        return MCGetCPMembersCodec.encodeResponse((List<Map.Entry<String, String>>) response);
+    }
+
+    @Override
+    public String getServiceName() {
+        return ManagementCenterService.SERVICE_NAME;
+    }
+
+    @Override
+    public Permission getRequiredPermission() {
+        return null;
+    }
+
+    @Override
+    public String getDistributedObjectName() {
+        return null;
+    }
+
+    @Override
+    public String getMethodName() {
+        return "mcGetCPMembers";
```

Comment:
Needs to be `getCPMembers`. We use the method name in the protocol definition. Same for other message tasks.

Reply:
My concern is potential collision with public client operations with the same name (`getCPMembers`) that could be introduced in future. Do you think I should add `mc` prefixes into client protocol for these methods, or it's fine to get rid of them here?

Reply:
That I don't know much about. Someone from the client team probably knows more. @asimarslan @sancar @ihsandemir what do you think?

Reply:
We are trying to match the method name to method name that user calls. If there is no User API, then the correct choice is not really well defined. In that sense, putting `mc` prefix to avoid a future collision is a good choice. 

Even if we are trying to match the names to protocol, I am not aware of it. @asimarslan What do you think ?

Reply:
As the correct approach is not certain, I've removed prefixes from `#getMethodName`s.